main.js

```diff
@@ -73,7 +73,9 @@ async function fetchNewPosts() {
 
    if(currentTimestampId > lastProcessedPostId && lastProcessedPostId != 0) {
       const text = removeHtmlTags(item.object.content);
-      postToBluesky(text);
+      await postToBluesky(text);
+      lastProcessedPostId = newTimestampId;
+      await saveLastProcessedPostId();
```

Comment:
Note that this doesn't technically have to use `await` here, since `saveLastProcessedPostId()` use the sync API, but an extra `await` doesn't hurt anything.